docs/rails/rails-engine-integration.md

```diff
@@ -14,21 +14,10 @@ s.add_dependency 'react_on_rails', '~> 6'
 ```ruby
 require "react_on_rails"
 ```
-+ In your `lib/tasks/<engine_name>_tasks.rake`:
-```ruby
-Rake.application.remove_task('react_on_rails:assets:compile_environment')
-
-task 'react_on_rails:assets:compile_environment' do
-  path = File.join(YourEngineName::Engine.root, 'client')
-  sh "cd #{path} && #{ReactOnRails.configuration.build_production_command}"
-end
-``` 
 ## In the project including your engine
 
 Place `gem 'react_on_rails', '~> 6'` before the gem pointing at your engine in your gemfile.
 
-This is necessary because React on Rails attaches itself to the rake assets:precompile task. It then uses a direct path to cd into client, which will not exist in the main app that includes your engine. Since you'll always be precompiling assets in the parent app, this will always fail. The workaround then, is to remove the task and replace it with one that goes into your Engine's root. The reason you have to include the react on rails gem before your engine is so that the `react_on_rails:assets:compile_environment` task is defined by the time your engine gets loaded to remove it.
-
```

Comment:
Found this while looking for where `assets:precompile` was mentioned.

Removed this because it's out of date: the task is `assets:webpack` instead of `assets:compile_environment` & cding to `client` is no longer hard coded. [prepend_cd_node_modules_directory](https://github.com/shakacode/react_on_rails/blob/9ef6b326213480d0022db42710e5517cad0317bc/lib/react_on_rails/utils.rb#L133) cd's to node_modules_location, which is `Rails.root` by default.

I can move the changes to this file to another PR if you'd prefer.